azure-pipelines.deprecate-package.yml

```diff
@@ -15,26 +15,41 @@ parameters:
 
 variables:
   - group: 'Github and NPM secrets'
-  - template: .devops/templates/variables.yml
-    parameters:
-      skipComponentGovernanceDetection: false
```

Comment:
Removed the use of this `variables` template because it [explicitly declares](https://github.com/microsoft/fluentui/blob/07e5161854990eb375bda399f344107142c27649/.devops/templates/variables.yml#L48) `dryRun` thus causing the condition to skip deprecation step to fail

`dryRun` mode set to `true` before running pipeline but npm deprecate step still runs 
<img width="822" alt="image" src="https://github.com/user-attachments/assets/d7b1bc69-f4b3-4910-a62a-dceb6d480a7d" />

Reply:
`dryRun` flag works as expected after changes
<img width="681" alt="image" src="https://github.com/user-attachments/assets/7a9b2e35-3b2a-43ce-999e-6228f7a37de2" />

Reply:
nice find ! wondering if what about other pipelines that rely on that variable ? which seems to be always force do `false` I guess we need to change to parameters for this instead of explicitly defined variable ( like https://github.com/microsoft/fluentui-contrib/pull/281/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R6 ) ?